### ISSUE TITLE
on Waiting For game to initallize stops working

### ISSUE THREAD

**Issue by jsienkiewicz13** [KuromeSan/MCEELoginSkip#4]
pls fix that bug
that bug cant load on Waiting for game to initallize

**Comment by jsienkiewicz13** [comment]
or i need to run it for adminstrator

**Comment by jsienkiewicz13** [comment]
still not works

**Comment by KuromeSan** [comment]
are you using the right mee.ptr for the version you have?

**Comment by therealalex5677** [comment]
I'm having the same problem, it gets stuck waiting for the game to initialize

**Comment by mrpenguinb** [comment]
> 
> 
> I'm having the same problem, it gets stuck waiting for the game to initialize

Did you change the address lines in the "mee.ptr" file to the right version? By default it isn't the latest.

**Comment by therealalex5677** [comment]
> > I'm having the same problem, it gets stuck waiting for the game to initialize
> 
> Did you change the address lines in the "mee.ptr" file to the right version? By default it isn't the latest and is probably incorrect. Also check not just what version number the game is, but also if it is **32-bit or 64-bit**.

Just did this and it worked, thank you!

**Comment by shroomsel** [comment]
Hello, I have the same "Waiting for the game to initialize" problem but I've tried everything above and it still doesn't work

**Comment by mrpenguinb** [comment]
> 
> 
> Hello, I have the same "Waiting for the game to initialize" problem but I've tried everything above and it still doesn't work

Education Edition was recently updated to version 1.17.30.5, so the game is currently unplayable at the moment unless you didn't update.

**Comment by shroomsel** [comment]
> > Hello, I have the same "Waiting for the game to initialize" problem but I've tried everything above and it still doesn't work
> 
> Education Edition was recently updated to version 1.17.30.5, these are the new values: New pointer path for 1.17.30.5: `03619D90 > 8 > 58 > C0 > 288 > 20 > A8 > 10` However it doesn't work for me at the moment either :(

Is there any way to make it work? I was using 1.17.30.2 before and used the pointer paths and it didn't work

**Comment by mrpenguinb** [comment]
> > Education Edition was recently updated to version 1.17.30.5, these are the new values: New pointer path for 1.17.30.5: `03619D90 > 8 > 58 > C0 > 288 > 20 > A8 > 10` However it doesn't work for me at the moment either :(
> 
> Is there any way to make it work? I was using 1.17.30.2 before and used the pointer paths and it didn't work

Currently I can't get the latest pointer path to work, so unfortunately I can't help you any further :(

**Comment by shroomsel** [comment]
> > > Education Edition was recently updated to version 1.17.30.5, these are the new values: New pointer path for 1.17.30.5: `03619D90 > 8 > 58 > C0 > 288 > 20 > A8 > 10` However it doesn't work for me at the moment either :(
> > 
> > 
> > Is there any way to make it work? I was using 1.17.30.2 before and used the pointer paths and it didn't work
> 
> I'm having the same issue as you. I don't know of any other method currently.

Did it work for you before 1.17.30.5?

**Comment by mrpenguinb** [comment]
> 
> 
> > > > Education Edition was recently updated to version 1.17.30.5, these are the new values: New pointer path for 1.17.30.5: `03619D90 > 8 > 58 > C0 > 288 > 20 > A8 > 10` However it doesn't work for me at the moment either :(
> > > 
> > > 
> > > Is there any way to make it work? I was using 1.17.30.2 before and used the pointer paths and it didn't work
> > 
> > 
> > I'm having the same issue as you. I don't know of any other method currently.
> 
> Did it work for you before 1.17.30.5?

Yes, 1.17.30.2 worked for me before updating. The main maintainer of this repo hasn't added the latest pointer path officially.

**Comment by shroomsel** [comment]
> > > I'm having the same issue as you. I don't know of any other method currently.
> > 
> > 
> > Did it work for you before 1.17.30.5?
> 
> Yes, 1.17.30.2 worked for me before updating. It turns out the other pointer path was for the 64-bit version. This is the 32-bit path for 1.17.30.5: `2CBA414 > 0 > 80 > C > 64 > 0 > 2E4 > 8`
It actually worked! Thank you so much.

**Comment by antibot128** [comment]
I will continue this issue.
Before this issue mceeloginskip worked perfectly. But when I turn off and then turn on the game mceeloginskip dont work. Nothing help me. After that I reboot my PC and turn on the game again. But mceeloginskip works perfectly again.
I think, problem of this issue is problem of OS (I have win7 x64).
